xml_inscriptions/transcribed/PA.Phil.UP.UM.L.L-1033-11.xml

```diff
@@ -51,8 +51,8 @@
                                         <depth>2</depth>
                                     </dimensions>
                                 </support>
-                                <condition ana="#complete">
-                                    <p/>
+                                <condition ana="#complete.broken">
+                                    <p>reconstituted from 2 pieces.</p>
```

Comment:
Note that you don't need periods here; really the only places that they're necessary are in the free commentary.